planner/core/point_get_plan.go

```diff
@@ -331,60 +332,74 @@ func getNameValuePairs(nvPairs []nameValuePair, expr ast.ExprNode) []nameValuePa
 		}
 		return nvPairs
 	} else if binOp.Op == opcode.EQ {
-		colName, ok := binOp.L.(*ast.ColumnNameExpr)
-		if !ok {
-			return nil
-		}
 		var d types.Datum
-		switch x := binOp.R.(type) {
-		case *driver.ValueExpr:
-			d = x.Datum
-		case *driver.ParamMarkerExpr:
-			d = x.Datum
```

Comment:
just add one more line `param=x` is not okay?

Reply:
It should be OK. Seems like previous code cannot handle `const = col` case, so I made this change.

Reply:
Oh i got it